features/draft_environment.feature

```diff
@@ -11,14 +11,14 @@ Feature: Draft environment
     Then I should be on the case study page
     And the page should contain the draft watermark
 
-  @draft
+  @draft @high
```

Comment:
hmm, there's no api in content store to expose searching/filtering by `format`, so I can't think of a way of doing that, unless the smokey tests would ssh onto a `backend` machine and interrogate the specialist publisher database directly.

Maybe it's a case of 'this will do for now', and if/when the document disappears we can try to make the test more robust, or remove it from the monitoring checks. By that point the publishing API work may be more advanced and we may have more options for addressing this.

Open to suggestions if anyone has any ideas.